scss/tech.scss

```diff
@@ -0,0 +1,40 @@
+.tech {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: space-around;
+  background-color: $earls-green;
+  font-family: 'Roboto', sans-serif;
+  height: 300px;
+  width: 200px;
+  border-radius: 5px;
+  align-items: center;
+  padding: 0 10px 10px 0;
+
+  &::before {
+    content: ' ';
+    position: absolute;
+    border: 1px solid $earls-green;
+    border-radius: 5px;
+    width: inherit;
+    height: inherit;
+  }
+
+  .tech-name {
+    font-size: 18px;
+  }
+
+  .tech-relase {
+    font-weight: bold;
+  }
+
+  .tech-logo {
+    height: 100px;
```

Comment:
Same here, there are already defined sizes, look on the trello cards and use them as variables.